### ISSUE TITLE
@auth with Aurora serverless endpoints

### ISSUE THREAD

**Issue by sirceljm** [aws-amplify/amplify-cli#3751]
**Which Category is your question related to?**

Api

**Amplify CLI Version** 

You can use `amplify -v` to check the amplify cli version on your system
4.16.1

**What AWS Services are you utilizing?**

Api, DynamoDb, Cognito, Storage

**Provide additional details e.g. code snippets**

I have a project where I successfully generated DynamoDB tables with @auth directive in the graphql.schema file. I would also like to include some tables from my Aurora Serverless DB cluster like described in the docs: https://aws-amplify.github.io/docs/cli-toolchain/graphql#relational-databases I can access the data from the table but I haven't seen anywhere how to define `@auth` permissions in this case.

Does amplify cli support this right now? Do I need to write custom resolvers?

**Comment by Graig123git** [comment]
@sirceljm Did you figure this out?

**Comment by sirceljm** [comment]
@Graig123git No, I quess just have to wait for the feature to get implemented.

**Comment by Graig123git** [comment]
I think its important to get this feature in, @sirceljm are u using custom resolvers to check auth or did you use another medium? right now i am using lambda resolvers to check if a user is authorize to access the aurora db models  but am wondering if there is a better way to do this at the moment.

**Comment by julien-tamade** [comment]
@sirceljm Was this ever resolved?

**Comment by cookmscott** [comment]
Would be great to get either: `@auth` directive working with GraphQL + Aurora Serverless with Amplify CLI ... OR Some documentation on how to set user read/write permissions to endpoints.

**Comment by Graig123git** [comment]
@cookmscott did you find a solution ?

**Comment by cookmscott** [comment]
@Graig123git no I did not. It looks like you need to create your own resolvers, which also has little to no documentation. TBH as a solo dev, I wasn't comfortable using Amplify with AuroraDB + GraphQL so I changed my backend due to lack of support and clear direction from AWS team.
